targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/trng_api.c

```diff
@@ -17,6 +17,10 @@
 #include "analogin_api.h"
 #include "analogin_ext.h"
 
+#ifdef CONFIG_MBED_ENABLED
```

Comment:
What are these new #ifdefs enabling that wasn't enabled before?

Reply:
Our platform has its own std libraries that are defined under platform_stdlib.h, some of these define are modified to support our specific realtek platform and are different from the original stdlib.h hence we have this define enabled here.